Fixed #28343 -- Add an OS chooser for docs command line examples.

ramiro opened the pull request:
Stopped suggesting usage of Git Shell on Windows. Using it doesn't
actually make things simpler, e.g. it isn't compatible with Python's
venv (already shipped with Python >= 3.5) so a special case directing
Windows users to install virtualenv was needed anyway.

Add a custom `console` Sphinx directive which provides a better,
multi-platform UI for CLI examples on the HTML output. Use it where
applicable.

Fixes #28343.

timgraham commented on docs/_theme/djangodocs/static/console-tabs.css:
```diff
@@ -0,0 +1,46 @@
+@import url("http://netdna.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css");
```
I guess we might want to include a copy of this so that the docs work locally without an internet connection?

ramiro replied:
Good catch. Done.

ramiro commented:
I've addressed the issue pointed out re: Bundling and using a local copy of Font Awesome.

Also, after reviewing all the ``.. code-block:: console`` occurrences in the documentation again, found:
* The GeoDjango tutorial CLI examples can also be converted to use the new directive.
* The uWSGI installation CLI example doesn't need to be shown for Windows and it's not available on that platform.

And fixed them.

timgraham commented:
I think the license file for the font should also be included.
"Does the full OFL license text always need to accompany the font?"
 http://scripts.sil.org/cms/scripts/page.php?item_id=OFL-FAQ_web#68092c0f

Update to the latest version of font awesome, 5.0.4?
